src/client/containers/view-batch-detail-page/view-batch-detail-page.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import { withKnobs } from '@storybook/addon-knobs';
+import BatchDetailPage from './view-batch-detail-page.component';
+
+export default {
+  component: BatchDetailPage,
+  title: 'view batch details page',
+  // decorators: [withKnobs],
```

Comment:
try not to leave too many comments around the code. You have all the code in github history if you ever want to go back to it.